handbook/technical-overview/api-design-guide/rest-response.md

```diff
@@ -74,22 +75,20 @@ For creating a resource
 For updating a existing resource
 
 - `200` should be returned after a successful execution, when there is a need for content in the response.
-- `204` should be returned after a successful execution, as usually there is no need for content in the response.
+- `204` should be returned after a successful execution, as usually there is no need for content in the response. It should also be used when the resource is not found or the user does not have permission to update it.
```

Comment:
This is not great, that we can't tell apart if the operation was successful or the resource not found.

I'd like to remove the first sentence and require that PUT and PATCH return a resource with 200 similar to POST.

Reply:
I agree